policy/README.md

```diff
@@ -109,18 +89,6 @@ Policies will be returned in order of effective date (see schema below), with pa
 
 `provider_id` is an implicit parameter and will be encoded in the authentication mechanism, or a complete list of policies should be produced. If the Agency decides that Provider-specific policy documents should not be shared with other Providers (e.g. punitive policy in response to violations), an Agency should filter policy objects before serving them via this endpoint.
 
-#### Geographies
-
-Endpoint: `/geographies/{id}`  
-Method: `GET`  
-`data` Payload: `{ geographies: [] }`, an array of GeoJSON `Feature` objects.
-
-##### Query Parameters
-
-| Name         | Type      | Required / Optional | Description                                    |
-| ------------ | --------- | --- | ---------------------------------------------- |
-| `id`         | UUID      | Optional    | If provided, returns one geography object with the matching UUID; default is to return all geography objects.               |
-
```

Comment:
it’s my understanding that the geographies services is not coming until MDS 1.1, so it seems premature to remove this from the 1.0 spec. should this section stick around until that is released? https://github.com/openmobilityfoundation/mobility-data-specification/pull/499

Reply:
Geography is just being refactored out of Policy.  AFAIK that's still on track for 1.0.

Reply:
Should that be part of this diff, though? Let us know if we’re missing something!

Reply:
Definitely not! Looks like some merging happened and this PR needs a refresh.  Thanks @jrheard @heatherbuletti

Reply:
Fixed by @thekaveman